content/blog/twitter-data-download-explanation.en.md

```diff
@@ -0,0 +1,903 @@
+{
+    "title": "Explaining Twitter's Data Download Feature.",
+    "type": "blog",
+    "url": "twitter-data-download-explanation",
+    "aliases": [
+    	"twitter-data",
+    	"gdpr-request"
+    ],
+    "date": "2019-10-17T16:58:19+02:00",
+    "description": "Twitter provides the option to download the user's personal data from their settings . This feature gives easier access to the user. In this, we explain the process to get the data as well as the meaning of the data provided.",
+    "featured_image": "/card-icons/Twitter_Logo_Blue.svg",
+    "tags": [ "twiiter", "gdpr", "data", "download","export" ],
+    "authors": [ "xndr" ],
+    "blog_like": true
+}
+
+<img class="offset-image offset-image-left" src="/card-icons/Twitter_Logo_Blue.svg" style="height: 200px;">
+
+Twitter is one of the top social media sites of our time. It has a high number of daily users and popular celebrities, sportstars and politicians use it as a method of communication with their fan base. It also serves as the source of daily news and entertainment for a lot of users. Using the insights from the world of hashtags, Twitter along with other brands and influencers have created an entirely new field of marketing. 
+
+Twitter collects the users personal data to create and serve ads to them. For this purpose, it collects relevant user data and stores it to determine user interest and likes. This data can be a treasure trove of information on any frequent user. If the user opts in, the data is also shared with Twiiter also stores a profile on any non logged-in user as well to serve ads based on their browsing habits. This is done on the device only and this data changes once the user logs in. The data stored is used to infer identity and serve ads as well as can be shared with other partners of Twitter.
+
+If you go to [Twitter's settings page](https://twitter.com/settings/your_twitter_data) for user data without any user being logged in, you'll see a page that looks a bit like this.
+
+<img class="offset-image offset-image-left" src="/img/blog/twitter-data-explanation/non-login-twitter-data-1.png" style="height: 500px; margin-top: 50px; margin-bottom: 100px;"> <img class="offset-image offset-image-right" src="/img/blog/twitter-data-explanation/non-login-twitter-data-2.png" style="height: 500px; margin-top: 50px; margin-bottom: 100px;"> 
+
+
+<a id="twitter-logged-in-procedure">     
+## The Procedure to get the Data Download from Twitter(Logged in user)
+</a>
+
+1. Go to your Account settings by clicking on the more icon in the navigation bar, and selecting Settings and privacy from the menu.
```

Comment:
In order to make menu items clear, I would make them italic: `*Settings and privacy`. This applies to all menu items and buttons.